src/Common/src/CoreLib/System/Runtime/CompilerServices/AsyncIteratorMethodBuilder.cs

```diff
@@ -42,7 +42,7 @@ public struct AsyncIteratorMethodBuilder
         /// <param name="stateMachine">The state machine instance, passed by reference.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void MoveNext<TStateMachine>(ref TStateMachine stateMachine) where TStateMachine : IAsyncStateMachine =>
-#if CORERT
+#if CORERT || MICROSOFT_BCL_ASYNCINTERFACES_PACKAGE
```

Comment:
Shoot, I meant to revert all changes to this file, but apparently I didn't push that commit.  Will do so in a subsequent PR.

Reply:
Could you please also remove the CORERT ifdef while you are on it? (I will close my PR to remove it then.)

Reply:
Ok, will do.